models/articles.model.js

```diff
@@ -50,14 +50,24 @@ const checkIdExists = (article_id) => {
     .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
     .then((result) => {
       if (result.rowCount === 0) {
-        return Promise.reject({ status: 404, msg: "Article not found" });
+        return Promise.reject({ status: 404, msg: "status 404: Article not found" });
       }
     });
 };
 
+const insertComment = (body, username, article_id) => {
+  if(body === '' || username === ''){
+     return Promise.reject({ status: 404, msg: "status 404: Article not found" });
+    } else {
```

Comment:
This model looks great! I do wonder whether this error message is correct for the if statement though (?)